### PR TITLE
Correct code

### PR DESCRIPTION
**Pull request by kuehma02**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,24 @@
-node_modules
```
Comment (kuehma02) [inline .gitignore]: I basically used the autogenerated .gitignore and the just added the backend/node_modules

File: backend/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+ATLAS_URI=mmongodb+srv://kuehma02:project-2020@cluster0-o9rln.azure.mongodb.net/test?retryWrites=true&w=majority
```
Comment (kuehma02) [inline backend/.env]: SHOOT THIS SHOULD NOT END UP HERE. We might have to add another line in the .gitignore. When i did this before I ended up using this link: http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

Comment (kuehma02) [reply backend/.env]: Review this

**Comment by kuehma02** [comment]
To set up this code, I used this tutorial: https://medium.com/@beaucarnes/learn-the-mern-stack-by-building-an-exercise-tracker-mern-tutorial-59c13c1237a1
I ended at the Create Exercise Component, which we can do later on. 

To get the app to deploy, I added the web.config file, which came from this article: https://medium.com/@to_pe/deploying-create-react-app-on-microsoft-azure-c0f6686a4321

I think that's everything for now

We have some conflicting files, and I'm not sure how/why those are happening. Feel free to request changes if needed before merging.

**Comment by kuehma02** [comment]
After this gets merged, we need to switch the repo that this comes from in azure.

File: backend/.env (hunk 2 above)
Comment (rublan01) [reply backend/.env]: Okay so should that be removed before we merge this in?

Comment (rublan01) [reply backend/.env]: Like is it a security concern to expose our database endpoint?

Comment (kuehma02) [reply backend/.env]: I think we can merge it in and then remove it, but I'm not sure how to take it out without just tossing the entire commit.

Comment (kuehma02) [reply backend/.env]: But yeah we don't want this sitting in our repo

Comment (rublan01) [reply backend/.env]: If you add it to the gitignore, run those commands in the link you sent, and then push to your branch again, it should update the PR!

**Comment by rublan01** [comment]
It also looks like there's going to be merge conflicts with our base branch. Might be good to do a PR to clear the repository (delete all old folders) and push that up to master, then add in the new files...merge conflicts aren't usually super fun haha

**Comment by rublan01** [comment]
> It also looks like there's going to be merge conflicts with our base branch. Might be good to do a PR to clear the repository (delete all old folders) and push that up to master, then add in the new files...merge conflicts aren't usually super fun haha

The have conflicts because we have files with the same name and completely different contents, and git just compares the files that are changed with the stuff hosted on the base branch online, it doesn't actually do a lot of deleting, so unless the names are different, it won't see them as different files. But if we delete them, commit up that they were deleted, and then readd the new files (with the same names as they had) I think we should be fine

**Pull request closed without merging**